libvast/vast/detail/tracepoint.hpp

```diff
@@ -36,18 +36,18 @@
 
 // # Overview
 //
-// A USDT (userspace software-defined tracepoint/userspace dynamic tracepoint)
-// is a code instrumentation mechanism provided by the linux kernel to allow
-// tracing software to measure and account specific developer-defined events in
-// user space code and libraries.
+// A USDT (userspace statically-defined tracepoint) is a code instrumentation
+// mechanism provided by the linux kernel to allow tracing software to measure
```

Comment:
I think the mechanism originates from DTrace on Solaris. Not sure if the Folly implementation works on other OSes though.

Reply:
We don't run on Solaris, so for VAST the linux kernel will provide the feature ;) (also the details are different, with dtrace you had a separate `tracepoints.d` file defining your tracepoints and dtrace itself would need to be integrated into the build pipeline generate an object file out of that which can be linked into the final binary, whereas here everything is directly generated inline. Or so they say in the manual, not that I've ever used dtrace personally.)

I've changed the paragraph a bit to include the historical context.